docs/posts/overview-of-ballerina-0.982.0.md

```diff
@@ -0,0 +1,327 @@
+```ballerina
+function queryChanges(string host, int port = 8080, string… params) returns float {
+
+}
+```
+Invoke the function using ballerina `run` as follows:
+```cmd
+ballerina run --printreturn currency:queryChanges localhost -port=8181 high day
+```
+The function invocation results in a value assignment as follows:
+    - `host` ← `"localhost"`
+    - `port` ← `8181`
+    - `params` ← `["high", "day"]`
+
+When the `--printreturn` flag is set, the return value is printed to the standard out stream.
+
+### New Documentation Syntax
+
+Old syntax:
+```ballerina
+documentation {
+    Adds parameter `x` and parameter `y`.
+    P{{x}} one thing to be added
+    P{{y}} another thing to be added
+    R{{}} the sum of them
+}
+function add(int x, int y) returns int { return x + y; }
+```
+
+New syntax:
+```ballerina
+# Adds parameter `x` and parameter `y`.
+# + x - one thing to be added
+# + y - another thing to be added
+# + return - the sum of them
+function add(int x, int y) returns int { return x + y; }
+```
+
+Note: Ballerina does not support the old documentation syntax with the new release. 
+
+### Reorder Documentation in Resources
+
+Earlier, documentation was added after annotation attachments. Now the documentation needs to be added before the 
+annotation attachment.
+
+Old syntax:
+```ballerina
+service<http:Service> update_token bind { port: 9295 } {
+
+    @http:ResourceConfig {
+        methods:["GET"]
+    }
+    # Updates the access token.
+    #
+    # + caller - Endpoint
+    one_px_image(endpoint caller) {
+
+    }
+}
+```
+
+New syntax:
+```ballerina
+service<http:Service> update_token bind { port: 9295 } {
+
+    # Updates the access token.
+    #
+    # + caller - Endpoint
+    @http:ResourceConfig {
+        methods:["GET"]
+    }
+    one_px_image(endpoint caller) {
+
+    }
+}
+```
+
+### Changes to Record/Object Field Syntax
+
+Old syntax:
+```ballerina
+type foo record {
+    string a,
+    string b,
+};
+
+type bar object {
+    string a,
+    string b,
+};
+```
+
+New syntax:
+```ballerina
+type foo record {
+    string a;
+    string b;
+};
+
+type bar object {
+    string a;
+    string b;
+};
+```
+
+### New Anonymous Functions Syntax
+
+The syntax of anonymous functions has been changed to resemble normal function definitions.
+
+Old syntax:
+```ballerina
+var lambda = (int b) => (int) {
+    int x = b * b;
+    return x;
+};
+```
+
+New syntax:
+```ballerina
+var lambda = function (int b) returns int {
+    int x = b * b;
+    return x;
+};
+```
+
+A new “Arrow Function Expression” is introduced as a simple alternative to anonymous functions that only have a return 
+statement in the body block. Syntax is as follows:
+```ballerina
+function (int, string) returns string lambdaVar = (param1, param2) => param2.toUpper();
+```
+
+### Removal of Implicit Cast from `int` to `float`
+
+We need to explicitly define a `float` with a decimal point now.
```

Comment:
Minor change:
Now it is necessary to explicitly define a `float` with a decimal point.